assets/src/userTestGroups.ts

```diff
@@ -0,0 +1,12 @@
+import appData from "./appData"
+
+function getTestGroups(): string[] {
```

Comment:
For what it's worth we usually declare TypeScript functions with `const` rather than `function` in most of the codebase.

Reply:
As for exporting I'd say it's fine to leave private for now and we can export it later if need be.

Reply:
> For what it's worth we usually declare TypeScript functions with `const` rather than `function` in most of the codebase.

I'm just curious, can you elaborate on why?

I understand that with `const` we get errors on reassignment, but we loose function hoisting, which doesn't matter in this case (and due to modules), but it seems strange to me that const functions would be preferred over the language keyword, but I feel like I'm missing something.